.github/workflows/scripts/check-collector-module-version.sh

```diff
@@ -19,6 +19,7 @@
 # as a dependency.
 #
 
+# shellcheck disable=SC1091
```

Comment:
Could you add a comment here?

Reply:
What are the reasons why we might have no access to this file? I don't understand how that would be possible if the file is in the repository. Maybe shellcheck needs to be configured differently?